### PR TITLE
CHAOS-1907: adds new free plan topic

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to Preview**](https://649187e5b9a3c3207b19309a--harness-developer.netlify.app/docs/chaos-engineering/get-started/learn-more-free-plan)

### CODE REVIEW

File: docs/chaos-engineering/get-started/learn-more-free-plan.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Harness CE free plan
+sidebar_label: Harness CE free plan
+sidebar_position: 4
+---
+
+Harness is the industry’s first software delivery platform to use AI to simplify your DevOps processes—CI, CD, GitOps, Feature Flags, Cloud Costs, Chaos Engineering and much more. 
+
+Harness is built on Litmus core, and offers a free plan in the hosted (SaaS) version of Harness Chaos Engineering (CE) for Litmus core community members. Once logged in, you can use Harness CE SaaS to execute your chaos scenarios in the cloud. Harness manages all your chaos control plane components, while the chaos execution plane components exist on your Kubernetes cluster as chaos infrastructure.
```
Comment (ajeshbaby) [inline docs/chaos-engineering/get-started/learn-more-free-plan.md:9]: Harness is built --> Harness CE(Harness Chaos Engineering) is built

File: docs/chaos-engineering/get-started/learn-more-free-plan.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Harness CE free plan
+sidebar_label: Harness CE free plan
+sidebar_position: 4
+---
+
+Harness is the industry’s first software delivery platform to use AI to simplify your DevOps processes—CI, CD, GitOps, Feature Flags, Cloud Costs, Chaos Engineering and much more.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/learn-more-free-plan.md:7]:
```suggestion
Harness is the industry’s first software delivery platform to use AI to simplify your DevOps processes—CI, CD, GitOps, Feature Flags, Cloud Costs, Chaos Engineering, and much more. 
```

File: docs/chaos-engineering/get-started/learn-more-free-plan.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+Harness is the industry’s first software delivery platform to use AI to simplify your DevOps processes—CI, CD, GitOps, Feature Flags, Cloud Costs, Chaos Engineering and much more. 
+
+[Harness](https://harness.io) offers a free hosted LitmusChaos for those who are trying to get started with LitmusChaos, and also for those who are getting started with Chaos Engineering (CE) in general. The Harness CE free plan is a free-forever plan that has features equivalent to LitmusChaos, and also bundles Harness platform features such as  RBAC and hosted logging—all for free.
+
+Sign up at [app.harness.io](https://app.harness.io) to get started with a free plan for either the hosted Harness CE or the hosted LitmusChaos. 
+
+:::info important
+The Harness CE free plan replaces [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). New signups are not being accepted at [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). Users on that platform are advised to move to the Harness CE free plan before August 31, 2023. For free support on migration, contact [Harness](https://harness.io) Support.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/learn-more-free-plan.md:14]:
```suggestion
The Harness CE free plan replaces [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). New signups are not being accepted at [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). Users on that platform are advised to move to the Harness CE free plan before August 31, 2023. For free support on migration, contact [Harness Support](mailto:support@harness.io).
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm

File: docs/chaos-engineering/get-started/learn-more-free-plan.md (hunk 3 above)
Comment (StrawberryPad) [reply docs/chaos-engineering/get-started/learn-more-free-plan.md:14]: They actually want new users to go to harness.io, so I need to leave as is.

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Pull request merged**